test/issue1003-check-empty-ld-flgs.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+cd $(CURR_DIR)/issue1003-check-empty-ld-flags
+
+# Fix includes changes to the argument generation for all
+# three compilers, test them all.
+$(DUB) build --compiler=dmd --force
+$(DUB) build --compiler=ldc --force
+$(DUB) build --compiler=gdc --force
```

Comment:
There are different test runners for the different compilers, trying to build all three of them on each setup may be the cause of that error.

Reply:
Changed it to just run it once with the `DC` envar.